src/inverseRobot.js

```diff
@@ -25,6 +25,20 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const robotFixed = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+    const newProperty = key;
+
+    if (robot[newProperty] in robotFixed) {
+      return null;
+    }
```

Comment:
It would be better to use `hasOwnProperty` method here.

Why? This will be explained further in the program in `Classes` topic